ntoskrnl/include/internal/amd64/ke.h

```diff
@@ -476,6 +476,13 @@ KiSetTrapContext(
     _In_ PCONTEXT Context,
     _In_ KPROCESSOR_MODE RequestorMode);
 
+VOID
+NTAPI
+KiInitializePcr(IN PKIPCR Pcr,
+                IN ULONG ProcessorNumber,
+                IN PKTHREAD IdleThread,
+                IN PVOID DpcStack);
```

Comment:
I am not super engaged about that, so I just fingerpoint to the fact that no SAL2 is used here. I won't reject for that sole reason though.

Reply:
To minimize changes, let's not SAL2'ify here because the original function in the .c file is still using old-style SAL.

Reply:
fine for me. Resolved.